test/cases/api.sh

```diff
@@ -683,10 +683,6 @@ function createReqFileGCP() {
   # - max 62 characters
   # - must be a match of regex '[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?|[1-9][0-9]{0,19}'
   #
-  # use sha224sum to get predictable 56 characters long testID without invalid characters
-  GCP_TEST_ID_HASH="$(echo -n "$TEST_ID" | sha224sum - | sed -E 's/([a-z0-9])\s+-/\1/')"
```

Comment:
These lines should not have been deleted, since the the value is used for other resource names, besides the image name.